Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Remove this file. We don't include it in clusterdock anymore.

Reply:
Done.